src/components/ImportRecordsFlow/stepTwo/styled/input.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+const StyledInput = styled.input`
+    position: absolute !important;
+    margin: -1px !important;
+    border: 0 !important;
+    padding: 0 !important;
+    width: 1px !important;
+    height: 1px !important;
+    overflow: hidden !important;
+    clip: rect(0 0 0 0) !important;
+    text-transform: none !important;
+    white-space: nowrap !important;
```

Comment:
instead of style this use the Structural HiddenElement that already has this styles:
```<HiddenElement as="input" />```